app/views/application/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1><%= t('hello') %></h1>
\ No newline at end of file
+<section class="front-section row">
+	<div class="container">
+		<%= image_tag "front-welcome.png", class: "pull-right img-rounded img-front-welcome" %>
```

Comment:
In case of responsive layouts `img-responsive` is in need, depending on your intention, see the introducing comment